docs/settings.md

```diff
@@ -1,38 +1,38 @@
 # Settings Reference
 
-This document serves as developer documentation for using "Granular Settings". Granular Settings allow users to specify 
-different values for a setting at particular levels of interest. For example, a user may say that in a particular room 
-they want URL previews off, but in all other rooms they want them enabled. The `SettingsStore` helps mask the complexity 
+This document serves as developer documentation for using "Granular Settings". Granular Settings allow users to specify
+different values for a setting at particular levels of interest. For example, a user may say that in a particular room
+they want URL previews off, but in all other rooms they want them enabled. The `SettingsStore` helps mask the complexity
 of dealing with the different levels and exposes easy to use getters and setters.
 
 
 ## Levels
 
-Granular Settings rely on a series of known levels in order to use the correct value for the scenario. These levels, in 
+Granular Settings rely on a series of known levels in order to use the correct value for the scenario. These levels, in
 order of priority, are:
 * `device` - The current user's device
 * `room-device` - The current user's device, but only when in a specific room
 * `room-account` - The current user's account, but only when in a specific room
 * `account` - The current user's account
 * `room` - A specific room (setting for all members of the room)
-* `config` - Values are defined by the `settingDefaults` key (usually) in `config.json`
+* `config` - Values are defined by the `setting_defaults` key (usually) in `config.json`
 * `default` - The hardcoded default for the settings
 
-Individual settings may control which levels are appropriate for them as part of the defaults. This is often to ensure 
+Individual settings may control which levels are appropriate for them as part of the defaults. This is often to ensure
 that room administrators cannot force account-only settings upon participants.
 
 
 ## Settings
 
-Settings are the different options a user may set or experience in the application. These are pre-defined in 
+Settings are the different options a user may set or experience in the application. These are pre-defined in
 `src/settings/Settings.tsx` under the `SETTINGS` constant, and match the `ISetting` interface as defined there.
 
-Settings that support the config level can be set in the config file under the `settingDefaults` key (note that some 
+Settings that support the config level can be set in the config file under the `setting_defaults` key (note that some
 settings, like the "theme" setting, are special cased in the config file):
-```json
+```json5
 {
   ...
-  "settingDefaults": {
+  "setting_defaults": {
     "settingName": true
```

Comment:
```suggestion
    "setting_name": true
```

Reply:
The settings can still be camel case, and most are, unfortunately.